src/middlewared/middlewared/alembic/versions/22.12/2022-08-17_13-37_privilege.py

```diff
@@ -0,0 +1,66 @@
+"""Account privileges
+
+Revision ID: 79942334975f
+Revises: e3a81e1c2135
+Create Date: 2022-08-09 13:37:57.243188+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '79942334975f'
+down_revision = 'e3a81e1c2135'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('account_privilege',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('builtin_name', sa.String(length=200), nullable=True),
+    sa.Column('name', sa.String(length=200), nullable=False),
+    sa.Column('local_groups', sa.TEXT(), nullable=False),
+    sa.Column('ds_groups', sa.TEXT(), nullable=False),
+    sa.Column('allowlist', sa.TEXT(), nullable=False),
+    sa.Column('web_shell', sa.Boolean(), nullable=False),
```

Comment:
Do you need to initialize the allow list for the default administrators group to `*` ?